js_modules/dagster-ui/packages/app-oss/src/pages/index.tsx

```diff
@@ -25,22 +25,3 @@ export default function IndexPage() {
     </div>
   );
 }
-
-/**
- * Ignore hard navigation error (only happens in dev mode).
- */
-if (process.env.NODE_ENV === 'development' && typeof window !== 'undefined') {
-  const originalError = window.Error;
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-ignore
-  window.Error = function Error(...args) {
-    if (args[0]?.includes('Invariant: attempted to hard navigate to the same URL')) {
-      return;
-    }
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore
-    const err = originalError(...args);
-    Object.setPrototypeOf(err, window.Error.prototype);
-    return err;
-  };
-}
```

Comment:
We don't need this anymore and it was changing error behavior in dev.

Reply:
Worth making this its own PR since it's otherwise unrelated?

Reply:
it's related in that this ends up changing the prototype of the error which breaks an instanceof check in Antlr which causes my code not to work